src/comics/routes.rs

```diff
@@ -38,6 +39,7 @@ pub fn comics_router() -> Router<AppState> {
         .route("/:comic_id", put(update_comic))
         .route("/:comic_id", delete(delete_comic))
         .route("/:comic_id", get(get_comic))
+        .route("/by_slug/:slug/:username", get(get_comic_by_slug))
```

Comment:
if we are gonna go by
`/api/v1/users/:username/comics/by_slug/:comic_slug/chapters/:chapter_number`
should this be
`/api/v1/users/:username/comics/by_slug/:comic_slug`
?

Reply:
Yes, to keep consistency in the API

Reply:
later :tm: 
if code is good and logical, refactoring will be easy later.